doc/source/whatsnew/v2.1.0.rst

```diff
@@ -141,6 +141,7 @@ Other enhancements
 - :meth:`MultiIndex.sortlevel` and :meth:`Index.sortlevel` gained a new keyword ``na_position`` (:issue:`51612`)
 - :meth:`arrays.DatetimeArray.map`, :meth:`arrays.TimedeltaArray.map` and :meth:`arrays.PeriodArray.map` can now take a ``na_action`` argument (:issue:`51644`)
 - :meth:`arrays.SparseArray.map` now supports ``na_action`` (:issue:`52096`).
+- :meth:`arrays.BaseMaskedArray.take` handle non-na float as fill value (triggered by ``ufloat`` NaN from ``uncertainties`` package) (see `PR <https://github.com/pandas-dev/pandas/pull/53970>`_)
```

Comment:
Could you describe this in terms of a public API? (BaseMaskedArray is not public)

Also ```(:issue:`53970`)``` is the appropriate citation

Reply:
I removed the code to which this documentation was referring: https://github.com/pandas-dev/pandas/pull/53970/commits/b8083a9a2559d22e41991a1d4c41fa266b64dfd3#diff-dddb661834ccfb6fde724041184559470d50f52e56719bdfa9c0d5e7922f0a51

I will revert this patch.